src/Discord.Net.Commands/Builders/ModuleClassBuilder.cs

```diff
@@ -10,45 +10,59 @@ namespace Discord.Commands
 {
     internal static class ModuleClassBuilder
     {
-        private static readonly TypeInfo ModuleTypeInfo = typeof(IModuleBase).GetTypeInfo();
+        public static readonly TypeInfo ModuleTypeInfo = typeof(IModuleBase).GetTypeInfo();
 
-        public static async Task<IReadOnlyList<TypeInfo>> SearchAsync(Assembly assembly, CommandService service)
+        ///<summary>Value1 - ready types, Value2 - types dependant on Value1.</summary>
+        public static async Task<Tuple<IReadOnlyList<TypeInfo>, IReadOnlyList<TypeInfo>>> SearchAsync(Assembly assembly, CommandService service)
```

Comment:
value tuple

Reply:
Unable to use value tuples, for now. More in-depth: #1393